src/components/CMSForm/index.tsx

```diff
@@ -82,7 +82,23 @@ export const CMSForm: React.FC<{
           setHasSubmitted(true)
 
           if (confirmationType === 'redirect' && redirect) {
-            if (redirect.url) router.push(redirect.url)
+            const { url } = redirect
+
+            if (!url) return
```

Comment:
returning here will skip the `submitForm()` at the end, is this intended? (maybe add a comment for that)